www/js/incident/post-trip-prompt.js

```diff
@@ -46,13 +46,14 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
 
     var reportNotifyConfig = {
       id: REPORT,
-      title: "Incident to report?",
+      title: "Did you use AccessMap to get your there?",
       text: reportMessage(ionic.Platform.platform()),
       icon: 'file://img/icon.png',
       smallIcon: 'res://ic_mood_question.png',
       sound: null,
-      actions: actions,
-      category: REPORT_INCIDENT_TEXT,
+      // actions: actions,
+      actions: [],
+      // category: CHOOSE_MODE_TEXT,
```

Comment:
I am not sure this will work without a category. Make sure to test on both iOS and android.
Original change is
https://github.com/e-mission/e-mission-phone/commit/129d4bbe43af63d044f28afe2659aee290e9a125